client/components/Error.tsx

```diff
@@ -0,0 +1,16 @@
+import { FormikErrors } from "formik";
+import React from "react";
+
+interface ErrorProps {
+	message: string | string[] | FormikErrors<any> | FormikErrors<any>[];
+}
+
+const Error = (props: ErrorProps) => {
+	return (
+		<div className="error__message">
```

Comment:
why do you need a div here?
This can be omitted and the styles can be applied to the container as well.

In fact, you can create a generic styled container and pass a prop called isError and can set the styles accordingly - like a success toast, error toast, warning toast, etc as per your requirement.